agent/containers/images/Dockerfile.base.j2

```diff
@@ -0,0 +1,22 @@
+# {{ distro_image_name }} pbench-agent base image
+FROM docker.io/library/{{ distro_image }}
+
+# Install the appropriate pbench repository file for {{ distro_image_name }}.
+COPY ./{{ pbench_repo_file }} /etc/yum.repos.d/pbench.repo
+
+# Install the pbench-agent RPM, which should have all its dependencies enumerated;
+# ... and make sure we have a proper pbench-agent.cfg file in place;
+# ... and finally, ensure the proper pbench-agent environment variables are setup.
+RUN \
+{% if distro_image.startswith('centos') %}
+    {{ pkgmgr }} install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-{{ distro_image.split(':', 1)[1] }}.noarch.rpm && \
+{% endif %}
+{% if distro_image == 'centos:7' %}
+    {{ pkgmgr }} install -y centos-release-scl-rh centos-release-scl && \
+    {{ pkgmgr }} install -y --enablerepo=centos-sclo-rh rh-python36 && \
+{% endif %}
+    {{ pkgmgr }} install -y {% if distro_image == 'centos:8' %}--enablerepo PowerTools {% endif %}--enablerepo copr-pbench pbench-agent && \
+    {{ pkgmgr }} -y clean all && \
+    rm -rf /var/cache/yum && \
+    cp -a /opt/pbench-agent/config/pbench-agent.cfg.example /opt/pbench-agent/config/pbench-agent.cfg && \
```

Comment:
Confused, why do we copy the `cfg.example` file to actual `cfg` file?

Reply:
Because for the 0.69 RPMs we don't provide a `pbench-agent.cfg` by default.